api/src/config.rs

```diff
@@ -1194,11 +1200,11 @@ fn default_work_dir() -> String {
     ".".to_string()
 }
 
-pub fn default_batch_size() -> usize {
-    128 * 1024
+fn default_user_io_batch_size() -> usize {
+    1024 * 1024
```

Comment:
Any analysis for this change?

Reply:
No analysis, but just keep the value as what is hard coded in the project currently, 1M. Nothing changed actually. See line 293 `batch_size: RAFS_DEFAULT_CHUNK_SIZE`

https://github.com/dragonflyoss/nydus/blob/f55d2c948f85e22041bda18d0f3c8de11ca4b8b7/storage/src/cache/fscache/mod.rs#L265-L294